samples/metrics_monitor/src/cttmetrics_i915_pmu.cpp

```diff
@@ -304,7 +347,14 @@ cttStatus CTTMetrics_PMU_Init()
     if (CTT_ERR_NONE != status)
         return status;
 
-    if (0 == perf_init(&g_ctx.pm, num_configs, configs)) {
+    g_ctx.gem_fd = drmOpenRender(128);
```

Comment:
That's still limited to Intel gfx being the first card on the system, right?

Reply:
I just tested "drmOpen("i915", NULL);". t works